cmd/subnetcmd/list.go

```diff
@@ -75,47 +76,58 @@ func listSubnets(cmd *cobra.Command, args []string) error {
 		}
 	}
 
-	for _, f := range files {
-		if strings.HasSuffix(f.Name(), constants.SidecarSuffix) {
-			carName := strings.TrimSuffix(f.Name(), constants.SidecarSuffix)
-			// read in sidecar file
-			sc, err := app.LoadSidecar(carName)
-			if err != nil {
-				return err
-			}
+	for _, s := range subnets {
+		// this shouldn't happen but let's be safe
+		if !s.IsDir() {
+			continue
+		}
+		subnetDir := filepath.Join(app.GetSubnetDir(), s.Name())
+		files, err := os.ReadDir(subnetDir)
+		if err != nil {
+			return err
+		}
+		for _, f := range files {
+			if f.Name() == constants.SidecarFileName {
+				carName := s.Name()
+				// read in sidecar file
+				sc, err := app.LoadSidecar(carName)
+				if err != nil {
+					return err
+				}
 
-			chainID := sc.ChainID
-			// for older sidecars, check in genesis if sidecar has
-			// no chainID set
-			if chainID == "" {
-				sc, err := app.LoadEvmGenesis(carName)
-				// ignore the error in this case: just leave it to ""
-				if err == nil {
-					chainID = sc.Config.ChainID.String()
+				chainID := sc.ChainID
+				// for older sidecars, check in genesis if sidecar has
+				// no chainID set
+				if chainID == "" {
+					sc, err := app.LoadEvmGenesis(carName)
+					// ignore the error in this case: just leave it to ""
+					if err == nil {
+						chainID = sc.Config.ChainID.String()
+					}
 				}
-			}
 
-			deployedLocal := "No"
-			if _, ok := deployedNames[sc.Subnet]; ok {
-				deployedLocal = "Yes"
-			}
-			deployedFuji := "No"
-			if _, ok := sc.Networks[models.Fuji.String()]; ok {
-				if sc.Networks[models.Fuji.String()].SubnetID != ids.Empty {
-					deployedFuji = "Yes"
+				deployedLocal := "No"
+				if _, ok := deployedNames[sc.Subnet]; ok {
+					deployedLocal = "Yes"
+				}
+				deployedFuji := "No"
+				if _, ok := sc.Networks[models.Fuji.String()]; ok {
+					if sc.Networks[models.Fuji.String()].SubnetID != ids.Empty {
+						deployedFuji = "Yes"
+					}
 				}
+				deployedMain := "N/A"
```

Comment:
Should we fix this to match Fuji?

Reply:
Ah I get that slipped through the cracks when we did the mainnet update